### PR TITLE
Apollo

### PR DESCRIPTION
**Pull request by kanersps**
ADD REAL DESCRIPTION HERE

This was created to leave comments for code

### CODE REVIEW

**Comment by kanersps** [comment]
@waltersajtos please remove the .idea directory and add it to gitignore

File: src/apollo-server.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+const LoggingPlugin = require("./plugins/logging");
+
+const typeDef = gql`
+    type Query
+`;
+
+const sources = [
+    WatchData,
+    CanvasData
+];
```
Comment (Typiqally) [inline src/apollo-server.js]: Please move this and following to a separate file to create more overview.

```ts
const CanvasData = require("./dataSources/canvas");
const WatchData = require("./dataSources/smartWatch");
const LoggingPlugin = require("./plugins/logging");
```

File: src/dataSources/canvas/canvas-api.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const {RESTDataSource} = require('apollo-datasource-rest');
+
+class CanvasAPI extends RESTDataSource {
+    willSendRequest(request) {
+        console.log(this.context.Authorization)
```
Comment (Typiqally) [inline src/dataSources/canvas/canvas-api.js]: This log is probably here for debugging purposes, but please remove before merging

File: src/plugins/logging.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const loggingPlugin = {
```
Comment (Typiqally) [inline src/plugins/logging.js]: Is this still used?

File: src/dataSources/smartWatch/watch-api.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+const {RESTDataSource} = require('apollo-datasource-rest');
+
+const StressData = require('./stress-data');
```
Comment (Typiqally) [inline src/dataSources/smartWatch/watch-api.js]: Maybe move this to a mock directory to make it clearer that it is mock data

**Review by Typiqally — CHANGES_REQUESTED** [review]
Very good work so far! I'm not sure whether it is possible with the time remaining, but I would like to see the code base migrated to TypeScript instead of JavaScript to ensure that the project stays modern.

**Pull request closed without merging**